lockpick-libzfs-abi.sh

```diff
@@ -0,0 +1,210 @@
+#! /usr/bin/env bash
+
+# Try to "lockpick" the settings relevant for libzfs.jar on this host OS
+# Note that the "correct answer" may change across OS updates... as well
+# as libzfs.jar evolution. Requires sources, "mvn" and JDK at this time,
+# and you may have to allow tests in pom.xml (set "<skip>" to "false").
+# Note it is likely to leave around Java coredump files, or at least logs
+# of those with stack traces.
+#
+# TODO: Add an option to libzfstest to run just a certain native routine.
+#
+# Copyright (C) 2017 by Jim Klimov
```

Comment:
Oh, I've didn't check. If it is CDDL, it is fine to keep it. I am just not a big fan of adding the Copyright text without specifying the license in a whatever way. Otherwise it looks like "all rights reserved, but IANAL